ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -62,4 +67,17 @@
     private CCDDocument sealedClaimDocument;
     private String features;
     private CCDYesNoOption migratedFromClaimStore;
+    private LocalDate docUploadDeadline;
```

Comment:
could we have all below elements in one of complex object?

Reply:
This is a good question, I actually thought that we are kind of  “polluting” the case with a lot of objects which could be contained in a complex type. But I am no CCD wizard like Sanchita, so she will know if it's possible :)